bin/upload-dist-versioned.sh

```diff
@@ -96,6 +98,11 @@ function copy_package_dist() {
         --metadata-directive REPLACE --metadata md5="${md5}",gitsha="${GITSHA}"
 }
 
+function copy_staging_release_to_dist() {
+    echo "copying s3://${S3_BUCKET}/${STAGING_PREFIX}/${STAGING_RELEASE_DIR} to s3://${S3_BUCKET}/${PACKAGE_PREFIX}/${VERSION_TAG}"
```

Comment:
should this be STAGING_RELEASE_DIR or STAGING_RELEASE?

Reply:
Fixed.